core/src/test/java/org/elasticsearch/search/aggregations/metrics/tophits/InternalTopHitsTests.java

```diff
@@ -103,6 +114,38 @@ protected InternalTopHits createTestInstance(String name, List<PipelineAggregato
         return new InternalTopHits(name, from, requestedSize, topDocs, searchHits, pipelineAggregators, metaData);
     }
 
+    @Override
+    protected void assertFromXContent(InternalTopHits aggregation, ParsedAggregation parsedAggregation) throws IOException {
+        final SearchHits expectedSearchHits = aggregation.getHits();
+
+        assertTrue(parsedAggregation instanceof ParsedTopHits);
+        ParsedTopHits parsed = (ParsedTopHits) parsedAggregation;
+        final SearchHits actualSearchHits = parsed.getHits();
+
+        assertEquals(expectedSearchHits.getTotalHits(), actualSearchHits.getTotalHits());
+        assertEquals(expectedSearchHits.getMaxScore(), actualSearchHits.getMaxScore(), 0.0f);
+
+        final Comparator<SearchHit> comparator = Comparator.comparing(SearchHit::getId);
+        List<SearchHit> expectedHits = Arrays.asList(expectedSearchHits.getHits());
+        expectedHits.sort(comparator);
```

Comment:
That's a leftover, this is indeed not necessary. Thanks for catching it Christoph!